### PR TITLE
K19DKPA-136

### PR DESCRIPTION
**Pull request by EugeneLokotariev**

### CODE REVIEW

File: backend/src/main/java/edu/netcracker/backend/controller/ApproverCrudController.java [hunk 1]
```diff
@@ -71,6 +71,14 @@ public BigInteger getRecordsCount() {
         for (User user : users) {
             userDTOS.add(UserDTO.from(user));
         }
+
```
Comment (volo0415) [inline backend/src/main/java/edu/netcracker/backend/controller/ApproverCrudController.java]: Please, delete this changes. It's not a cool!
`for (User user : users) {
            userDTOS.add(UserDTO.from(user));
}`

File: backend/src/main/java/edu/netcracker/backend/controller/ApproverCrudController.java [hunk 2]
```diff
@@ -71,6 +71,14 @@ public BigInteger getRecordsCount() {
...
+
+        // bla bla bla
+
+        ///
+
+
+        ///
```
Comment (volo0415) [inline backend/src/main/java/edu/netcracker/backend/controller/ApproverCrudController.java]: bla bla bla bla

Comment (EugeneLokotariev) [reply backend/src/main/java/edu/netcracker/backend/controller/ApproverCrudController.java]: bla bla bla

File: backend/src/main/java/edu/netcracker/backend/controller/ApproverCrudController.java (hunk 1 above)
Comment (EugeneLokotariev) [reply backend/src/main/java/edu/netcracker/backend/controller/ApproverCrudController.java]: You were wrong

**Pull request closed without merging**